src/generator/operations.ts

```diff
@@ -505,6 +509,13 @@ function isArrayOfRFC1123(schema: Schema): boolean {
 function createProtocolResponse(client: string, op: Operation, imports: ImportManager): string {
   const info = <OperationNaming>op.language.go!;
   const name = info.protocolNaming.responseMethod;
+  if (!op.responses) {
+    let text = `${comment(name, '// ')} handles the ${info.name} response.\n`;
```

Comment:
I think this block could be moved down a bit so the first two lines aren't duplicated.